etc/bankdefs/hipo4/event.json

```diff
@@ -495,7 +515,8 @@
             {"name":"cx",      "type":"F", "info":"direction cosline of the track at the detector surface"},
             {"name":"cy",      "type":"F", "info":"direction cosline of the track at the detector surface"},
             {"name":"cz",      "type":"F", "info":"direction cosline of the track at the detector surface"},
-            {"name":"path",    "type":"F", "info":"pathlength of the track to the detector surface from the DOCA point (cm)"}
+            {"name":"path",    "type":"F", "info":"pathlength of the track to the detector surface from the DOCA point (cm)"},
+            {"name":"edge",    "type":"F", "info":""}
```

Comment:
the info field should be filled. What about:
"signed distance between the track position at the detector surface and the closest surface edge (cm)"

the comment applies to all the banks.